src/scss/application.scss

```
@@ -126,6 +126,180 @@
+  position: absolute;
+  z-index: 1000;
+  max-width: 320px;
+  padding: 6px 10px;
+  background-color: #0b0c0c;
+  color: #ffffff;
+  font-family: $govuk-font-family;
+  font-size: 14px;
+  line-height: 1.3;
+  border-radius: 3px;
+  pointer-events: none;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)